source/utils/urlUtils.py

```diff
@@ -0,0 +1,18 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2020-2024 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+
+def _valueToSamePage(value: str, constantIdentifier: str) -> bool:
```

Comment:
@coderabbitai do you think this function can be improved by using urllib.parse? Could you also generate unit tests for this function?